types/webpack-env/index.d.ts

```diff
@@ -305,9 +305,8 @@ declare var __webpack_require__: any;
  * The internal chunk loading function
  *
  * @param chunkId The id for the chunk to load.
- * @param callback A callback function called once the chunk is loaded.
  */
-declare var __webpack_chunk_load__: (chunkId: any, callback: (require: __WebpackModuleApi.RequireLambda) => void) => void;
+declare var __webpack_chunk_load__: (chunkId: any) => Promise<any>;
```

Comment:
```suggestion
declare var __webpack_chunk_load__: (chunkId: any) => Promise<void>;
```

Return type appears to be `Promise<void>` -> https://webpack.js.org/api/module-variables/#__webpack_chunk_load__-webpack-specific